include/deal.II/base/function_parser.h

```diff
@@ -26,10 +26,7 @@
 #include <vector>
 #include <map>
 
-namespace mu
-{
-  class Parser;
-}
+#include <muParser.h>
```

Comment:
I'm trying to invistigate the problem. I've found that there is smth strange with tbb::enumerable_thread_specific. The minimal example:
https://gist.github.com/resetius/063750a05b0a2f8f6bd4a0f99a7158ca
`std::vector<std::vector<A> >` compiles ok
`tbb::enumerable_thread_specific<std::vector<A> >` fails on cl